### ISSUE TITLE
PrometheusExporter

### ISSUE THREAD

**Issue by iDustbin** [creepymonster/GlucoseDirect#571]
Hi, would it be possible to include an PrometheusExporter to export all values to an selfhosted Prometheus instance, cloud ?

**Comment by DAcodedBEAT** [comment]
I don't think it would be a good idea to directly expose a prometheus exporter from this application, since iOS apps typically do not expose a webserver (probably due to iOS being able to stop the server at any time, especially when inactive - and when it's active, it's actively using the radios which would contribute to battery drain).


If you want to remotely monitor your data, how about integrating with [NightScout](http://www.nightscout.info/) to have a centralized location for your glucose data? (this is already integrated into the application)  From some light googling, it looks like someone exported from NightScout to prometheus via https://github.com/dabear/nightscout_exporter, so that could be an option if you still want to export prometheus metrics from somewhere.

_disclaimer_: I am not a maintainer of this repository so I can't speak for the development direction for this project.
